index.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Microverse</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap" rel="stylesheet">
+  <script
+      src="https://kit.fontawesome.com/e520ead38a.js"
+      crossorigin="anonymous"
+    ></script>
+</head>
+<body>
+  <header>
+    <nav>
+      <div class="top">
+        <i class="fa-brands fa-facebook-f social"></i>
+        <i class="fa-brands fa-twitter social"></i>
+        <ul class="top-list">
+          <li class="top-items">English</li>
+          <li class="top-items">MyPage</li>
+          <li class="top-items">Logout</li>
+        </ul>
+      </div>
+      <div class="main-nav">
+        <img src="./images/microverseLogo.png" class="logo" alt="Microverse Logo">
+        <ul class="menu-mobile" id="menu">
+          <li class="mobile-menu-list" onclick="closeMenu()"><a href="./about.html">About</a></li>
+          <li class="mobile-menu-list" onclick="closeMenu()"><a href="#">Program</a></li>
+          <li class="mobile-menu-list" onclick="closeMenu()"><a href="#">Join</a></li>
+          <li class="mobile-menu-list" onclick="closeMenu()"><a href="#">Sponsor</a></li>
+          <li class="mobile-menu-list" onclick="closeMenu()"><a href="#">News</a></li>
+          <i class="fa-solid fa-xmark close" onclick="closeMenu()"></i>
+        </ul>
+        <ul class="menu">
+          <li class="menu-list"><a href="./about.html">About</a></li>
+          <li class="menu-list"><a href="#">Program</a></li>
+          <li class="menu-list"><a href="#">Join</a></li>
+          <li class="menu-list"><a href="#">Sponsor</a></li>
+          <li class="menu-list"><a href="#">News</a></li>
+          <button type="button" class="button">CC Campaign</button>
+        </ul>
+      </div>
+      <i class="fa-solid fa-bars open" onclick="openMenu()"></i>
+    </nav>
+  </header>
+  <main>
+    <section class="body">
+      <article class="main-text">
+        <p class="intro">"Hello! Sharing world"</p>
+        <h1 class="title">Become an international software developer, from anywhere.</h1>
+        <article class="information">
+          <p>Gain the skills you need to get hired. Learn online, connect to life-changing opportunities. Pay $0 until you land a job!</p>
+        </article>
+      </article>
```

Comment:
- [ ] Let's stick to the design as much as possible. Please update the font size. It looks a bit bigger on the template.
Kindly add the last part which is the date on the design. You can add your own custom content as long as it is a visual copy of the design. 😎️

**Your Design**
![Screenshot from 2023-01-10 10-22-56](https://user-images.githubusercontent.com/17243325/211495946-ffda5453-18d1-41f6-91c4-ae33897bb248.png)

**Template**
![Screenshot from 2023-01-10 10-16-33](https://user-images.githubusercontent.com/17243325/211496027-400d3d60-bebf-4f78-992a-94a439aabb10.png)

Reply:
Fixed.